docs/intro.rst

```diff
@@ -65,11 +65,13 @@ Pywbem also provides a command line utility:
   repository (e.g. on a WBEM server). See :term:`DSP0004` for a definition of
   MOF.
 
-The related `pywbemtools project`_ provides this command line utility:
+The related `pywbemtools project`_ provides these command line utilities:
 
-* `pywbemcli`_ -
-  A client-side command line interface for a WBEM server, supporting a command
-  line mode and an interactive (repl) mode.
+* `pywbemcli`_ - A client-side command line interface for a WBEM server,
+  supporting a command line mode and an interactive (repl) mode.
+
+* `pywbemlistener`_ - A command that runs and manages WBEM indication listeners
```

Comment:
builddoc flags this line with:

/home/kschopmeyer/pywbem/pywbem39/pywbem/docs/intro.rst:73: WARNING: Unknown target name: "pywbemlistener".

Reply:
DONE - added the link target to both intro.rst and README.rst.